apps/src/blockly/addons/cdoUtils.js

```diff
@@ -89,3 +89,29 @@ export function getField(type) {
   }
   return field;
 }
+
+/**
+ * Returns xml string for a block, after removing any next block
+ * connected to it. Used by context menu options and shortcuts.
+ * This should not be used elsewhere as it is scheduled for deletion
+ * once we implement @blockly/plugin-cross-tab-copy-paste
+ * @param {block}
+ * @returns {string}
+ */
+export function getSingleBlockXml(block) {
```

Comment:
I know this is a time-sensitive PR, but just curious about why we need to add this helper function to achieve this effect when Google Blockly's copy and cut behavior already behaves like this. Is it possible to use their `copy` which doesn't include [next blocks](https://github.com/google/blockly/blob/92efdb91d12281a412aac4e87701471b80406393/core/block_svg.ts#L944) as we use their `dispose`?

Reply:
ALICE! You were so right. Check out how much cleaner this is now that I'm using the built-in `toCopyData` and `pasteBlock_` (https://github.com/google/blockly/blob/34634ea57cd43dac9028f1a28e4beb761e60eef6/core/workspace_svg.ts#L1400).